### ISSUE TITLE
Unable to download 1.5.0 from https://civicrm.org/extensions/upcoming-birthday-report

### ISSUE THREAD

**Issue by mrickly** [systopia/de.systopia.birthdays#47]
Clicking on the download button on the extension page https://civicrm.org/extensions/upcoming-birthday-report fails with "the given path has multiple possibilities: #<Git::Ref:0x00007f2591eb6780>, #<Git::Ref:0x00007f2591eb60f0>".
Similarly, when I try to update from 1.5.0.-alpha1 from within CiviCRM, I get an error.

**Comment by bjendres** [comment]
Thanks for reporting, @mrickly. 

@jofranz, I believe you have recently released a couple of version here, does this ring a bell?

**Comment by mlutfy** [comment]
I updated the link on civicrm.org (but no idea why the link was wrong, I'll check if it happens again on future releases)

**Comment by bjendres** [comment]
> I updated the link on civicrm.org (but no idea why the link was wrong, I'll check if it happens again on future releases)

Thanks @mlutfy! 

@jofranz When you're back, could you still have a quick look to understand what happened?

**Comment by jofranz** [comment]
Can't seem to find a reason why this didn't work. Sometimes weird things happen with [docs-books](https://lab.civicrm.org/documentation/docs-books) when there is a force push, but this should not be the case for extension version listings here.

I only noticed "Björn" being the "Author" of "Birthdays" and we renamed it at some point from something like "Birthday xtension" to "Birthdays".

Let's see if if it happens again next time.

@mlutfy Thanks for fixing it!
